stable/kong/values.yaml

```diff
@@ -51,10 +51,19 @@ proxy:
   annotations: {}
   #  service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 
-  # HTTPS traffic on the proxy port
-  useTLS: true
-  servicePort: 8443
-  containerPort: 8443
+  # HTTP plain-text traffic
+  http:
+    enabled: false
```

Comment:
The comment got lost somewhere, let's enable HTTP and HTTPS traffic by default!

Reply:
Enabled and updated the README's default value.